mt-metadata/README.md

```diff
@@ -0,0 +1,27 @@
+# /mt-metadata/
+This [W3ID](https://w3id.org) provides a persistent URI namespace for mt-metadata resources.
+
+
+## Uses
+This namespace is used to provide URIs for the Metadata standards for Magnetotelluric Time Series Data  (mt_metadata) hosted in [Research Vocabularies Australia (RVA)](https://vocabs.ardc.edu.au).
+
+The project is documented on the [mt_metadata GitHub repository](https://doi.org/10.5281/zenodo.5605025) and the [mt_metadata documentation website](https://mt-metadata.readthedocs.io/en/latest/index.html).
+
+
+## Redirects
+- https://w3id.org/mt-metadata -> Research Vocabularies Australia (RVA)
+
+
+## Contact
+This space is administered by:  
+
+**NCI Data Collections**   
+National Computational Infrastructure (https://nci.org.au)
+Canberra, Australia
+<datacollections.nci@anu.edu.au>
+GitHub: [@NCI-datacollections](https://github.com/NCI-datacollections)
+
+**NCI Help**  
+National Computational Infrastructure (https://nci.org.au)
+Canberra, Australia
+<help@nci.org.au>
```

Comment:
to preserve apparently intended line breaks --
```suggestion
**NCI Data Collections**  
National Computational Infrastructure (https://nci.org.au)  
Canberra, Australia  
<datacollections.nci@anu.edu.au>  
GitHub: [@NCI-datacollections](https://github.com/NCI-datacollections)

**NCI Help**  
National Computational Infrastructure (https://nci.org.au)  
Canberra, Australia  
<help@nci.org.au>
```

Reply:
Thanks @TallTed. That looks fab.

Reply:
Note that you'll need to submit another PR to get these into place.